src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts

```diff
@@ -425,4 +429,32 @@ describe('EntryFieldsComponent', () => {
     expect(component.loadActiveEntrySubscription.unsubscribe).toHaveBeenCalled();
     expect(component.actionSetDateSubscription.unsubscribe).toHaveBeenCalled();
   });
+
+  it('The flag "update_last_entry_if_overlap" is added to the "newData" when feature flag "update-entries" is enabled for user', () => {
```

Comment:
Maybe this could be more descriptive: when feature ... enable for the user and running entry update is called

Reply:
Thanks Kelly for your comments, I will consider what you say